assets/src/edit-story/elements/media/scalePanel.js

```diff
@@ -45,6 +45,7 @@ const Container = styled.div`
   border-radius: 8px;
   padding: 3px ${HORIZONTAL_PADDING}px;
   margin-top: 8px;
+  box-shadow: 0 0 3px white;
```

Comment:
This should use a design-system color but awaiting confirmation that this is actually okay to add.